src/nodes/interceptor/eth_send_transaction.go

```diff
@@ -54,6 +56,13 @@ func (i *Interceptor) sendPrivateTx(ctx context.Context, msg *ethereum.SendTxMsg
 		msg.Data = new([]byte)
 	}
 
+	// extract aliases from PrivateFor
+	*msg.PrivateFor, err = placeholder.ReplaceAliases(ctx, i.aliases, *msg.PrivateFor)
+	if err != nil {
+		i.logger.WithError(err).Error("failed to replace aliases in privateFor")
+		return nil, errors.BlockchainNodeError(err.Error())
```

Comment:
`BlockchainNodeError` doesn't seem to be the correct error here.

Reply:
You're right, I think it's a copypasta…

Reply:
fixed in dca55e7